package.json

```diff
@@ -21,8 +21,9 @@
   "dependencies": {
     "@turf/turf": "^4.7.3",
     "bluebird": "^3.5.1",
-    "lodash": "^4.17.4",
     "express": "^4.16.2",
+    "lodash": "^4.17.4",
```

Comment:
We already removed lodash in one of the latest revisions, so I am guessing you are using an outdated version of our code...

Secondly, why are you adding `pinkie` here?

Reply:
I think I submitted my pull request before #133 was closed that should be why lodash is there. 

As far as pinkie goes, I was having an error when I tried npm install it said that pinkie was on 2.0.5 so I ran a npm install --save pinkie and the version went to 2.0.4 and my error went away